index.php

```diff
@@ -467,21 +467,23 @@
 
     <span id="translate-download" style="display:none;" hidden>Download elementary OS</span>
     <span id="translate-purchase" style="display:none;" hidden>Purchase elementary OS</span>
-    <div id="download-modal" class="dialog modal">
-      <img alt="Download elementary OS icon" src="images/icons/apps/48/system-os-installer.svg">
+
+    <dialog id="download-modal" class="dialog modal" aria-labelledby="download-modal-title">
```

Comment:
Using aria-labelledby to tie the heading to the dialog itself so it's clear what it's for, need to test screen reader behaviour, I think we may need to focus the dialog when it opens...